src/MetalPerformanceShaders/MPSCnnKernel.cs

```diff
@@ -0,0 +1,37 @@
+﻿#if XAMCORE_2_0 || !MONOMAC
+using System;
+using XamCore.Metal;
+using XamCore.Foundation;
+
+namespace XamCore.MetalPerformanceShaders {
+	public partial class MPSCnnBinaryConvolution {
+		public unsafe MPSCnnBinaryConvolution (IMTLDevice device, IMPSCnnConvolutionDataSource convolutionData, float [] outputBiasTerms, float [] outputScaleTerms, float [] inputBiasTerms, float [] inputScaleTerms, MPSCnnBinaryConvolutionType type, MPSCnnBinaryConvolutionFlags flags)
+			: base (NSObjectFlag.Empty)
+		{
+			fixed (void *outputBiasTermsHandle = outputBiasTerms) {
```

Comment:
minor same as above, this is really up to you :) if you want to change it, I think I find it easier to read that way so I do not have to right scroll that much on the `InitializeHandle` line
```csharp
fixed (void *outputBiasTermsHandle = outputBiasTerms)
fixed (void* outputScaleTermsHandle = outputScaleTerms)
fixed (void* inputBiasTermsHandle = inputBiasTerms)
fixed (void* inputScaleTermsHandle = inputScaleTerms)
	InitializeHandle (InitWith (device, convolutionData, (IntPtr) outputBiasTermsHandle, (IntPtr) outputScaleTermsHandle, (IntPtr) inputBiasTermsHandle, (IntPtr) inputScaleTermsHandle, type, flags));
```